msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/AcquireTokenCIAMTest.java

```diff
@@ -56,27 +67,90 @@ public LabUserQuery getLabUserQuery() {
         return query;
     }
 
-    @Ignore //Disabling temporary due to tenant bug
+    public static AuthenticationCallback successfulVerifyIssuerCallback() {
+        return new AuthenticationCallback() {
+            @Override
+            public void onSuccess(IAuthenticationResult authenticationResult) {
```

Comment:
nit: final to params here and everywhere

Reply:
updated